R/target_market_share.R

```diff
@@ -390,14 +390,65 @@ reweight_technology_share <- function(data, ...) {
 }
 
 aggregate_by_loan_id <- function(data) {
-  aggregate_columns <- c("id_loan", "loan_size_outstanding", "loan_size_credit_limit")
 
   data %>%
-    dplyr::group_by_at(setdiff(names(data), aggregate_columns)) %>%
+    group_by(
+      .data$level,
+      .data$loan_size_outstanding_currency,
+      .data$loan_size_credit_limit_currency,
+      .data$name_ald,
+      .data$sector_ald
```

Comment:
We don't have an object holding this, right? Maybe some `crucial_market_share_cols` or something like that sitting somewhere else? Just wondering.

Reply:
nay we do not afaik